packages/fx-core/src/core/index.ts

```diff
@@ -486,7 +487,8 @@ class CoreImpl implements Core {
                 return ok(null);
             };
 
-            const signinM365Callback = async (): Promise<Result<null, FxError>> => {
+            const signinM365Callback = async (args?: any[]): Promise<Result<null, FxError>> => {
+                this.ctx?.telemetryReporter?.sendTelemetryEvent(TelemetryEvent.LoginM365, getTriggerFromProperty(args));
```

Comment:
And we should have the following properties according to PM's spec: 
"component", "hashed-userid", "internal": "internal-alias"

Reply:
@chagong can you share the spec? I try to understand what these properties are.